matRad/dicom/@matRad_DicomImporter/matRad_interpDicomDoseCube.m

```diff
@@ -44,66 +43,65 @@
 
 % read the dosefile itself
 dosedata = dicomread(dosefile);
-dose.cube = double(dosedata);
-% dose.cube = single(dosedata);
+obj.importRTDose.dose.cube = double(dosedata);
+% importRTDose.dose.cube = single(dosedata);
 
 % give it an internal name
-dose.internalName = currDose{12};
+ % obj.dose.internalName = obj.currDose{12};%?????
```

Comment:
whats going on here ?

Reply:
I remember that we discussed this, and I still have no idea, what is this for

Reply:
id suggest removing it